sdk/cpp/core/src/xml_subtree_codec.cpp

```diff
@@ -271,20 +250,21 @@ static void check_and_set_node(Entity &entity, Entity *parent,
              to_string(xml_node->name), entity.yang_name);
   check_payload_to_raise_exception(entity, xml_node->name);
   auto child_name = to_string(xml_node->name);
-  if (xml_node->ns->href && xml_node->parent && xml_node->parent->ns->href) {
-    auto child_ns = to_string(xml_node->ns->href);
-    if (child_ns != to_string(xml_node->parent->ns->href)) {
-      string module_name;
-      auto capabilities_lookup_table = get_global_capabilities_lookup_tables();
-      auto it = capabilities_lookup_table.find(child_ns);
-      if (it != capabilities_lookup_table.end()) {
-        module_name = it->second.module;
-      } else {
-        module_name = child_ns.substr(child_ns.rfind("/") + 1);
-      }
-      child_name = module_name + ":" + child_name;
-    }
-  }
+  // if (xml_node->ns->href && xml_node->parent && xml_node->parent->ns->href) {
```

Comment:
should have been deleted? 
is this because we don't support namespaces for non-top-level nodes?